synapse/rest/media/v1/media_repository.py

```diff
@@ -78,16 +78,16 @@ def __init__(self, hs: "HomeServer"):
 
         Thumbnailer.set_limits(self.max_image_pixels)
 
-        self.primary_base_path = hs.config.media_store_path  # type: str
-        self.filepaths = MediaFilePaths(self.primary_base_path)  # type: MediaFilePaths
+        self.primary_base_path: str = hs.config.media_store_path
+        self.filepaths: MediaFilePaths = MediaFilePaths(self.primary_base_path)
```

Comment:
Another spot I'm surprised we need a type hint?

Reply:
Yeah they're automated changes, i'm more surprised that they're assigned there in the first place though. I'll remove :+1: